src/platform/utilities/oauth/crypto.js

```diff
@@ -8,8 +8,22 @@ const decimal2hex = dec => `0${dec.toString(16)}`.substr(-2);
  */
 export function sha256(plain) {
   // Transforms string into a Uint8Array (utf8)
+  if (!plain || plain.length === 0) {
+    throw new Error('input must be defined');
+  }
+
   const data = new TextEncoder().encode(plain);
-  return window.crypto.subtle.digest('SHA-256', data);
+  try {
+    // eslint-disable-next-line import/no-unresolved
+    const { webcrypto } = require('node:crypto');
+    return webcrypto?.subtle?.digest('SHA-256', data);
+  } catch (err) {
+    if (typeof window !== 'undefined' && window?.crypto?.subtle) {
+      return window?.crypto?.subtle?.digest('SHA-256', data);
+    }
+
+    throw new Error('crypto.subtle not available');
+  }
```

Comment:
Basically tries to use Node v22 `webcrypto` as primary and then Node v14.16.1 is the fallback.